roles/network/templates/hostapd/iiab-hotspot-on

```diff
@@ -9,18 +9,17 @@ echo -e "If you add Wi-Fi hardware, run 'cd /opt/iiab/iiab' then 'sudo ./iiab-ne
 echo -e "For details, see: https://github.com/iiab/iiab/pull/3179\n"
 exit 1
 {% else %}
-sed -i -e "s/^HOSTAPD_ENABLED.*/HOSTAPD_ENABLED=True/" {{ iiab_env_file }}
+sed -i "s/^HOSTAPD_ENABLED.*/HOSTAPD_ENABLED=True/" {{ iiab_env_file }}
 {% if wifi_up_down %}
 systemctl enable iiab-clone-wifi.service
 systemctl enable hostapd
 systemctl enable iiab-wifi-test.service
-echo -e "\nPlease reboot to activate hostapd feature.\n"
-exit 0
+#exit 0
 {% else %}
 {% if is_raspbian %}
 # just do what we have always done in hotspot-on
 cp -f /etc/hostapd/hostapd.conf.iiab /etc/hostapd/hostapd.conf
-sed -i -e "s/^#denyinterfaces/denyinterfaces/" /etc/dhcpcd.conf
+sed -i "s/^#denyinterfaces/denyinterfaces/" /etc/dhcpcd.conf
 # shut down wlan0 in case connected to network
 ip link set wlan0 down
 systemctl enable hostapd
```

Comment:
@jvonau 3 lines below this (on Line 29) `systemctl start hostapd` is run (if OS is RasPiOS).

Is there any hope for doing the same (i.e. running `systemctl start hostapd` within iiab-hotspot-on) on other OS's too — i.e. just below Line 44?

Separately: Should similar possibly be considered around Line 16 (which applies to all OS's) in the very separate case where `wifi_up_down` ?

Reply:
In short no, and the way that is written one can't use wifi_up_down False and install over wifi, the original reason for hotspot-on|off to begin with. You want to wear the chef's hat go ahead but I'm not your dishwasher.